examples/bzlmod/MODULE.bazel

```diff
@@ -0,0 +1,25 @@
+module(
+    name = "bzlmod_example",
+    repo_name = "bzlmod_example",
+)
+
+bazel_dep(name = "com_github_mvukov_rules_ros")
+# local_path_override just so that one can test this locally
+local_path_override(
+    module_name = "com_github_mvukov_rules_ros",
+    path = "../..",
+)
+# but normal usage would be to use a git override
+#git_override(
+#    module_name = "com_github_mvukov_rules_ros",
+#    remote = "https://github.com/mvukov/rules_ros.git",
+#    commit = "160940582fea35a2335d04183e0c540c1209d27e", # TODO update 
+#)
```

Comment:
My recommendation would be to use https://bazel.build/rules/lib/globals/module#archive_override with https://github.com/mvukov/rules_ros//archive/SHA_OF_COMMIT.zip as url.

The zip file to only download what you need and `archive_override` as you want to consume it via bzlmod